src/views/table.js

```diff
@@ -57,6 +57,18 @@ async function renderTable(tableNumber,playerNumber) {
           <button id="newCard">Deal a card (0)</button>
           <button id="collectCards">Collect cards & shuffle</button>
         </p>
+        <p>
```

Comment:
Is this used anywhere? I assume we want to show the number of cards in players hands. I didn't find it, but maybe I missed somethig.